spec/functional/matcher_errors_spec.rb

```diff
@@ -0,0 +1,45 @@
+describe "Error Messages" do
+  it "renders a helpful error message without root or object_root options" do
+    example = nil
+    group = RSpec.describe "user.rabl" do
+      include_context "user_context"
+      example = it{ expect(subject).to render_attribute(:guid).with_value("Imma derp derp") }
+    end
+    group.run
+    lines = RSpec::Core::Notifications::FailedExampleNotification.new(example).message_lines
```

Comment:
I couldn't find a better way to do this. I looked at how rspec tests rspec and came up with this solution.